third_party/terraform/tests/resource_app_engine_standard_app_version_test.go

```diff
@@ -127,6 +136,117 @@ resource "google_storage_bucket_object" "main" {
 }`, context)
 }
 
+func testAccAppEngineStandardAppVersion_vpcAccessConnector(context map[string]interface{}) string {
+	return Nprintf(`
+resource "google_project" "my_project" {
+  name = "tf-test-appeng-std%{random_suffix}"
+  project_id = "tf-test-appeng-std%{random_suffix}"
+  org_id = "%{org_id}"
+  billing_account = "%{billing_account}"
+}
+
+resource "google_app_engine_application" "app" {
+  project     = google_project.my_project.project_id
+  location_id = "us-central"
+}
+
+resource "google_project_service" "project" {
+  project = google_project.my_project.project_id
+  service = "appengine.googleapis.com"
+
+  disable_dependent_services = false
+}
+
+resource "google_project_service" "vpcaccess_api" {
+  project = google_project.my_project.project_id
+  service = "vpcaccess.googleapis.com"
+
+  disable_dependent_services = false
+}
+
+resource "google_vpc_access_connector" "bar" {
+	depends_on = [
+    google_project_service.vpcaccess_api
+  ]
+	project = google_project.my_project.project_id
+	name = "bar"
+	region = "us-central1"
+	ip_cidr_range = "10.8.0.0/28"
+	network = "default"
+}
+
+resource "google_app_engine_standard_app_version" "foo" {
+	depends_on = [
+		google_vpc_access_connector.bar
```

Comment:
If we add the interpolation on the name below we shouldn't need this depends_on

Reply:
fixed